docs/connectors/matrix.md

```diff
@@ -6,16 +6,18 @@ Maintained by [@SolarDrew](https://github.com/SolarDrew).
 
 ## Requirements
 
-To use this connector you will need to have a Matrix account, and login using your Matrix username (mxid) and password.
+To use this connector you will need to have a Matrix account, and login using your Matrix username (mxid) and access token. You can find the access token in the **Advanced** section of the **Help & About** tab in the **Settings** of the official <Riot.im> web client.
```

Comment:
This needs rewording. I think we should be recommending username + password first as it's the easiest way.